bitnami/postgresql/templates/configmap.yaml

```diff
@@ -15,7 +15,11 @@ data:
 {{- else if .Values.postgresqlConfiguration }}
   postgresql.conf: |
 {{- range $key, $value := default dict .Values.postgresqlConfiguration }}
-    {{ $key | snakecase }}={{ $value }}
+    {{- if kindIs "string" $value }}
+    {{ $key | snakecase }} = '{{ $value }}'
+    {{- else }}
+    {{ $key | snakecase }} = {{ $value }}
+    {{- end }}
```

Comment:
What do you think about change this?
change this lines
```yaml
postgresql.conf: |
{{- range $key, $value := default dict .Values.postgresqlConfiguration }}
    {{- if kindIs "string" $value }}
    {{ $key | snakecase }} = '{{ $value }}'
    {{- else }}
    {{ $key | snakecase }} = {{ $value }}
    {{- end }}
```
to 
```yaml
  postgresql.conf: |
    {{- include "common.tplvalues.render" ( dict "value" .Values.postgresqlConfiguration "context" $ ) | nindent 4 }}
```

Reply:
Sorry for the late response, had a busy week.
However, correct me if i'm wrong, but the common.tplvalues.render does not include the detection of strings and quoting them?
This would be the exact same problem as before.

Reply:
Ok, I saw that there is some kind of string detection, however this is not sufficient in order to have all values correctly transformed:

```yaml
postgresqlConfiguration:
  maxConnections: 1000
  idleInTransactionSessionTimeout: 2000
  listenAddresses: "*"
  sharedBuffers: 128MB
  dynamicSharedMemoryType: 'posix'
  maxWalSize: 1GB
  minWalSize: 80MB
  logTimezone: 'UTC'
  datestyle: 'iso, mdy'
  timezone: 'UTC'
  defaultTextSearchConfig: 'pg_catalog.english'
  ssl: "on"
  ssl_ca_file: "/opt/ngi/certs/root.crt"
  ssl_cert_file: "/opt/ngi/certs/server.crt"
  ssl_key_file: "/opt/ngi/certs/server.key"
```

results in:

```
  postgresql.conf: |
    datestyle: iso, mdy
    defaultTextSearchConfig: pg_catalog.english
    dynamicSharedMemoryType: posix
    idleInTransactionSessionTimeout: 2000
    listenAddresses: '*'
    logTimezone: UTC
    maxConnections: 1000
    maxWalSize: 1GB
    minWalSize: 80MB
    sharedBuffers: 128MB
    ssl: "on"
    ssl_ca_file: /opt/ngi/certs/root.crt
    ssl_cert_file: /opt/ngi/certs/server.crt
    ssl_key_file: /opt/ngi/certs/server.key
    timezone: UTC
```